udata/forms/fields.py

```diff
@@ -296,9 +303,7 @@ def process_formdata(self, valuelist):
         if valuelist and len(valuelist) > 1:
             self.data = valuelist
```

Comment:
BTW, I kept the MIN_TAG_LENGTH and MAX_TAG_LENGTH in fields.py because it is used here only, i.e. not by the harvesters. Do you think the harvesters should crop the tags when they have not the right length ? Or ignore them ? Or raise an error ?